front/app/containers/IdeasEditPage/IdeasEditForm.tsx

```diff
@@ -80,14 +70,6 @@ const IdeasEditForm = ({ ideaId }: Props) => {
     inputId: ideaId,
   });
 
-  useEffect(() => {
-    if (idea && authUser !== undefined && !granted) {
-      // TODO: Fix this the next time the file is edited.
-      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-      clHistory.replace(previousPathName || (!authUser ? '/sign-up' : '/'));
-    }
-  }, [idea, granted, previousPathName, authUser]);
```

Comment:
previousPathName is always defined (as soon as we can grab `location` in `App`) and, in all cases I checked, equal to the page we're on, so no redirecting was happening. Regardless, adding context and updating `App` state on every route change just for this one edge case is hard to justify even if it were working.